KUBEVIRT_LOCAL_TESTING.md

```diff
@@ -1,34 +1,52 @@
 # [kubevirtci](README.md): Testing kubevirt with locally provisioned cluster
 
-With the changes in place you also can directly execute [`make functest`](https://github.com/kubevirt/kubevirt/blob/master/docs/getting-started.md#testing) against a cluster locally with kubevirt that was provisioned using `kubevirtci`.
+After making changes to a kubevirtci provider, you should test it locally before publishing it.
+
+With the changes in place you can also directly execute [`make functest`](https://github.com/kubevirt/kubevirt/blob/master/docs/getting-started.md#testing) against a cluster locally with kubevirt that was provisioned using `kubevirtci`.
+
+Lets go on the required steps, starting by provisioning, all the way to `make functest`.
 
 Steps:
 
 ### kubevirtci: provision cluster locally
 
 ```bash
 cd .../kubevirtci
-cd cluster-provision/k8s/1.17.0
+cd cluster-provision/k8s/1.19
```

Comment:
in another PR: fix other instances of `git grep k8s/1....0/`

Reply:
https://github.com/kubevirt/kubevirtci/pull/578
didn't update the kind sriov, since the the provider still have 1.17.0 in its name all over.